mapper/tedge_mapper/src/mapper.rs

```diff
@@ -88,7 +88,7 @@ impl Mapper {
         Ok(())
     }
 
-    fn map(input: &[u8]) -> Result<Vec<u8>, thin_edge_json::ThinEdgeJsonError> {
```

Comment:
Wondering why you decided to add an extra level named `json` when the create name already has json in it.

Reply:
> Just not convinced by the new module name, but I not better suggestion. Why `json`?

Because it's JSON and it's already under thin edge crate, I would say it's better than thin edge json.

Reply:
> Wondering why you decided to add an extra level named `json` when the crate name already has json in it. Why not use `lib.rs` as the file name, which seems to be the standard?

Didier wanted to move this json part itself to a separate module. Also, I agree we should rename the thin-edge-json crate to thin-edge-message framework or something appropriate.